R/map.R

```diff
@@ -182,3 +182,41 @@ map_if <- function(.x, .p, .f, ...) {
   .x[sel] <- lapply(.x[sel], .f, ...)
   .x
 }
+
+#' Modify elements of a specified subset.
+#'
+#' @inheritParams map
+#' @param .at A character vector of names or a numeric vector of
```

Comment:
Oh hmmmmm, I wonder if there's a way we can integrate this with `map_if`? Maybe at least they should be documented together so it's easy to see the differences/similarities (i.e. `map_if()` is all about logicals, `map_at()` is all about position)